client/jetpack-connect/site-topic.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from 'components/formatted-header';
+import jetpackOnly from './jetpack-only';
+import MainWrapper from './main-wrapper';
+import SiteTopicForm from 'signup/steps/site-topic/form';
+import { loadTrackingTool } from 'state/analytics/actions';
+
+class JetpackSiteTopic extends Component {
+	componentDidMount() {
+		this.props.loadTrackingTool( 'HotJar' );
+	}
+
+	handleSubmit() {
+		// @TODO: implement saving logic
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<MainWrapper isWide>
+				<div className="jetpack-connect__step">
+					<FormattedHeader headerText={ translate( 'Tell us about your website?' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Tell us about your website' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Thanks @a8ci18n, great suggestion! Fixed in https://github.com/Automattic/wp-calypso/pull/30731/commits/5dc155f23b41bf3c057a234961494d6dc0439134